docs/_posts/murat-gunay/2021-11-04-classifierdl_use_sentiment_pipeline_tr.md

```diff
@@ -21,7 +21,7 @@ This pretrained pipeline identifies the sentiments (positive or negative) in Tur
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/1.Clinical_Named_Entity_Recognition_Model.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```

Comment:
I don't think this is correct. classifier_dl_use_sentiment_pipeline is not an NER model.